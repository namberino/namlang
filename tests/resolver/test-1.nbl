var a = "global";
{
    fun show_a()
    {
        print(a);
    }

    show_a(); // global
    var a = "block";
    show_a(); // global
}
