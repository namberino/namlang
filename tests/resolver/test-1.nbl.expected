Executing file: tests/resolver/test-1.nbl
global
global
